src/index.ts

```diff
@@ -127,7 +180,8 @@ function getEmit(
     // Emit Javascript
     const output = instance.languageService.getEmitOutput(filePath);
 
-    loader.clearDependencies();
+    // Remove original dependency on resourcePath and replace with rawFilePath
```

Comment:
I think maybe you missed the comments on outdated changes (probably me, not being an experienced GitHub user). But the gist is this:

The problem with calling `clearDependencies` is that it clears dependencies for the entire chain, not just the current loader. So if a loader has been run before ts-loader, any dependencies it has added (in the case of `ts-css-loader` this could be dependencies on `.css` files) will be cleared.

Normally, if there are no previous loaders in the chain, there would only be one file dependency: the ts-file being loaded (`resourcePath`).

I think what is really wanted is not to clear dependencies, but to replace the original dependency on `resourcePath` with a dependency on `rawFilePath`, which is defined like so:
```ts
const rawFilePath = path.normalize(this.resourcePath);
```

That's why I replaced `clearDependencies` with `removeDependencies` and `addDependency`.